jre/xccdf/java.xml

```diff
@@ -90,6 +90,11 @@ To configure the Java mandatory deployment setting, add or modify
 <tt>deployment.system.config.mandatory</tt> to equal <tt>true</tt>
 in <tt>/etc/.java/deployment/deployment.config</tt>.
 </description>
+<warning category="general">This recommendation is opposite of what is in DISA
```

Comment:
@dahaic rather than add this comment. Let's do a complete update of the JRE content and remove STIG deprecated products such as JRE6 and JRE7.

Reply:
@redhatrises JRE8 STIG is quite different so the changes will be (relatively) extensive - I'll do it on top of these commits, though, so I can reference fix in BZ. Is it ok?